composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "require-dev": {
         "psr/log": "~1.0",
-        "phpcr/phpcr-api-tests": "2.1.4",
+        "phpcr/phpcr-api-tests": "dev-clean@dev",
```

Comment:
change to 2.1.5 and jackalope/jackalope to 1.2.1 once https://github.com/phpcr/phpcr-api-tests/pull/163 is merged and we tagged a new version